src/Emulator.jl

```diff
@@ -385,22 +352,23 @@ elements on the main diagonal (i.e., the variances), we return the full
 covariance at each point, as a vector of length N_predicted_points, where 
 each element is a matrix of size output_dim × output_dim
 """
-function svd_reverse_transform_mean_cov(μ, σ2, 
-                                        decomposition::Union{SVD, Decomposition};
-					truncate_svd::FT=1.0) where {FT}
-    @assert ndims(μ) == 2
-    @assert ndims(σ2) == 2
+function svd_reverse_transform_mean_cov(
+    μ::AbstractMatrix{FT},
+    σ2::AbstractMatrix{FT}, 
+    decomposition::SVD
+) where {FT<:AbstractFloat}
+    @assert size(μ) == size(σ2)
     
     output_dim, N_predicted_points = size(σ2)
     # We created meanvGP = D_inv * Vt * mean_v so meanv = V * D * meanvGP
-    sqrt_singular_values= Diagonal(sqrt.(decomposition.S))
+    sqrt_singular_values = Diagonal(sqrt.(decomposition.S))
     transformed_μ = decomposition.V * sqrt_singular_values * μ
     
     transformed_σ2 = [zeros(output_dim, output_dim) for i in 1:N_predicted_points]
     # Back transformation
     
     for j in 1:N_predicted_points
-        σ2_j = decomposition.V * sqrt_singular_values * Diagonal(σ2[:,j]) * sqrt_singular_values * decomposition.Vt
+        σ2_j = decomposition.U * sqrt_singular_values * Diagonal(σ2[:,j]) * sqrt_singular_values * decomposition.Vt
```

Comment:
I think this should technically be V not U, as we are doing a coord transformation `(V*S^{1/2})*X*(V*S^{1/2})^T`,

Reply:
Thanks for catching this! It was a misunderstand on my part. Fixed in [e9e30bb](https://github.com/CliMA/CalibrateEmulateSample.jl/pull/132/commits/e9e30bb020863835aac117617adbd99435d0a2ad).